protobuf/types/execution.proto

```diff
@@ -114,4 +114,9 @@ message Execution {
   string price = 14 [
     (gogoproto.moretags) = 'hash:"name:14" validate:"coins"'
   ];
+
+  // blockHeight where the execution was included into blockchain.
+  int64 blockHeight = 15 [
+    (gogoproto.moretags) = 'hash:"-"'
```

Comment:
this should be part of the hash, no?

Reply:
If we do that, we cannot be sure that the hash calculated on the client or even during the processing will be the same. What happens if the execution is created on one block but in the mempool for 5 blocks.